modules/carto/src/layers/schema/carto-raster-tile-loader.ts

```diff
@@ -36,8 +36,12 @@ const CartoRasterTileLoader: LoaderWithParser = {
 };
 
 export type Raster = {
-  blockWidth: number;
-  blockHeight: number;
+  /** @deprecated Use blockSize. */
+  blockWidth?: number;
```

Comment:
I don't think any public API was ever using these, I would just remove them to avoid confusion. When we deploy this we will bump the map version, so the client should only talk to a server that returns `blockSize`